src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  let discount = 0;
```

Comment:
Prefer `const` over `let` for the `discount` variable since it's not being reassigned in different scopes but rather just once in the conditional blocks.